lib/new-admin/admin-server.js

```diff
@@ -53,6 +57,9 @@ apolloServer.applyMiddleware({
 // cors on app for /api/register endpoint.
 app.use(cors({ credentials: true, origin: devMode && 'https://localhost:3000' }))
 
+app.use('/id-card-photo', serveStatic(idPhotoCardBasedir, {index: false}))
+app.use('/front-camera-photo', serveStatic(frontCameraBasedir, {index: false}))
```

Comment:
This is a security issue since auth is being handled by graphql, needs to be further discussed.

Reply:
Got it. Is it a good idea to get the images from gql itself?

Reply:
I'm thinking we can either grab them from graphql, pull auth to the outside of graphql or anonymize it with random strings

Reply:
@liordino just look at this server side code and its already being handled like you're doing, so no harm done here. Image strings are random enough as it is.